policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/amount.yaml

```diff
@@ -1,21 +1,8 @@
 description: California provides the following foster youth tax credit amount, based on age.  
-brackets:
```

Comment:
why remove this? CA limits the credit to 18-25

Reply:
You mentioned that the age eligibility for CalEITC encompasses the FYTC age eligibility so there would be no differences in practice, and that's why I thought we could delete it. I wasn't very sure about whether the age limit is implicit or explicit. The legal code says this when explaining what's a qualified taxpayer: (c) For purposes of this section, the following definitions shall apply:
(1) "Qualified taxpayer," means an individual who satisfies all of the following:
(A) Has been allowed a tax credit under Section 17052 for the taxable year.
(B) Is 18 to 25 years of age, inclusive, as of the last day of the taxable year.
(C) Was in foster care while 13 years of age or older in an AFDC-FC placement, as described in Section 11402 of the Welfare and Institutions Code, including a tribally approved home, as defined in subdivision (r) of Section 224.1 of the Welfare and Institutions Code, or Approved Relative Caregiver Funding Program eligible placement, as described in Article 6 (commencing with Section 11450) of Chapter 2 of Part 3 of Division 9 of the Welfare and Institutions Code, by a Title IV-E agency, pursuant to a voluntary placement agreement or a juvenile court order.
(2) "Threshold amount" shall be twenty-five thousand dollars ($25,000). I know that it mentions about the age limit but I'm still not very sure if CalEITC encompasses this in practice.

Reply:
CalEITC age eligibility encompasses FYTC, not vice versa

Reply:
Ok. Does it mean that I only need to check the upper age limit (25) for FYTC? Because we already had the 18 as the minimum age for CalEITC eligibility.

Reply:
As always, represent the law in the code. That means 18-25.

Reply:
I see. I will add the age limit back.